those_who_play_tetris.md

```diff
@@ -169,7 +169,7 @@ Last year, Jilin Television invited him to another show. Also a variety show, al
 
 Modian later got a job at a cyber cafe as a counter clerk. Literally a clerk, the kind that collects fees and swipes ID cards. The reason for this job choice is simple, "the job as a clerk is not too busy and I get time to play games". Two months later, he left this job, and got to participate in recording sessions of another show. After the show, he joined a power plant as a maintenance crew that fixes whoever's broken computer at the site. The job is "a bit more busy", but it's a state-owned company, so the job is more stable, so still not bad.
 
-After the show gets more popular, some media started covering him. Some reporters saw that the "master of games" started doing "ordinary jobs", and wrote a story of "how a genius falls", and wrote down the job position as "security guard in the title. Modian was angry after seeing it, sating the media was writing bullshit and have no bottom line.
+After the show gets more popular, some media started covering him. Some reporters saw that the "master of games" started doing "ordinary jobs", and wrote a story of "how a genius falls", and wrote down the job position as "security guard in the title. Modian was angry after seeing it, sating the media was writing bullshit and have no bottom line".
```

Comment:
Help me move the quote after the "security guard", i.e. this pair of quotes should only cover the two words "security guard". I'm not sure how to edit someone else's PR, especially not on mobile lol

Reply:
also **sating** the media -> **saying** the media